jasypt/src/main/java/org/jasypt/intf/service/JasyptStatelessService.java

```diff
@@ -56,6 +57,10 @@ public JasyptStatelessService() {
         super();
     }
     
+    private String randomString = RandomStringUtils.randomAlphanumeric(12);
```

Comment:
@prakashiit  use [SecureRandom](https://docs.oracle.com/javase/7/docs/api/java/security/SecureRandom.html) for creating password

Reply:
Done!